.gitignore

```diff
@@ -1,46 +1 @@
-# Compiled class file
-*.class
-
-# Log file
-*.log
-
-# BlueJ files
-*.ctxt
-
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
-
-# Package Files #
-*.jar
-*.war
-*.nar
-*.ear
-*.zip
-*.tar.gz
-*.rar
-
-# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
-hs_err_pid*
-replay_pid*
-
-# Build artifacts
-target/
-build.log
-
-# Eclipse files
-.classpath
-.project
-.settings
-bin/
-
-# IntelliJ files
-/shelf/
-/workspace.xml
-.idea/
-*.iml
-
-# Mac files
-.DS_Store
-._*
-Footer
-/.history/
+.terraform*
```

Comment:
I think the Java .gitignore entries got deleted. This could impact the Java development team in their local environment.